v2/setup/mesos-master.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+source /etc/environment
+
+HOMEDIR=$(eval echo "~`whoami`")
+
+AWS_CREDS=""
+if [ ! -z $AWS_ACCESS_KEY ]; then
+    AWS_CREDS=" -e AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY \
+     -e AWS_SECRET_ACCESS_KEY=$AWS_SECRET_KEY "
+fi
+
+sudo docker run --rm \
+    -v ${HOMEDIR}:/data/ $AWS_CREDS behance/docker-aws-s3-downloader \
```

Comment:
@anshupande please change it - why would we keep a reference to the behance containers?